russian/light-fire/brake-the-wall/brake-the-wall.md

```diff
@@ -4,4 +4,6 @@
 
 Как вы будете спасаться от них?
 
-[Притворитесь, что вы тоже пчела и начнете жужжать.](bee-too/bee-too.md)
\ No newline at end of file
+[Притворитесь, что вы тоже пчела и начнете жужжать.](bee-too/bee-too.md)
+
+[Посмотрите в зеркало](i-am-the-bee.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.